.github/workflows/cwf-integ-test.yml

```diff
@@ -12,6 +12,11 @@ on:  # yamllint disable-line rule:truthy
       - 'v1.*'
     types:
       - completed
+  pull_request:
+    branches:
+      - master
+      - 'v1.*'
+    types: [opened, reopened, synchronize]
```

Comment:
I understand this is just temporal, right?

Reply:
yes

Reply:
Yes - I would like to remove this at the very last minute. Because without the cwf integ tests will not run on this branch.